deliverables/ruby_monk_problems/calculator.rb

```diff
@@ -1,9 +1,11 @@
+#Create a class Calculator, which performs addition and subtraction of two numbers at a time
+
 class Calculator
   def add(a, b)
    a + b
   end
 
   def subtract(a, b)
-   # your code here
+   a - b
   end
```

Comment:
Great, so this is an easy problem and perhaps you can rewrite this using Ruby's splat operator: https://endofline.wordpress.com/2011/01/21/the-strange-ruby-splat/